Maven Clean Install failed on com.zsmartsystems.zigbee.test

Using Maven 3.5.3 on Debian Stretch.
[ERROR] Failed to execute goal com.gavinmogan:codacy-maven-plugin:1.0.3:coverage (codacy-upload-coverage) on project com.zsmartsystems.zigbee.test: The parameters 'apiToken', 'projectToken' for goal com.gavinmogan:codacy-maven-plugin:1.0.3:coverage are missing or invalid

This can be ignored, but I will try and find a way to avoid the failure. The issue is just that you don’t have the API codes to upload the coverage results to codacy.com.

> On 8 Apr 2018, at 21:50, ncpe20012003 <notifications@github.com> wrote:
> 
> Using Maven 3.5.3 on Debian Stretch.
> [ERROR] Failed to execute goal com.gavinmogan:codacy-maven-plugin:1.0.3:coverage (codacy-upload-coverage) on project com.zsmartsystems.zigbee.test: The parameters 'apiToken', 'projectToken' for goal com.gavinmogan:codacy-maven-plugin:1.0.3:coverage are missing or invalid
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/zsmartsystems/com.zsmartsystems.zigbee/issues/272>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AA_kQ8rkPx7qz54ZGFeD2aJVYTTAAGd2ks5tmngegaJpZM4TLs5R>.
>

--- a/com.zsmartsystems.zigbee.test/pom.xml
+++ b/com.zsmartsystems.zigbee.test/pom.xml
@@ -72,7 +72,7 @@
                 <executions>
                     <execution>
                         <id>codacy-upload-coverage</id>
-                        <phase>verify</phase>
+                        <phase>deploy</phase>
                         <goals>
                             <goal>coverage</goal>
                         </goals>

